src/internal/starlark/starlark.go

```diff
@@ -0,0 +1,232 @@
+// Package starlark runs Pachyderm-specific starlark programs.  It's recommended to import this as
+// "ourstar" when also using go.starlark.net/starlark in the same package.
+package starlark
```

Comment:
Kind of annoying to have to use a named import throughout. The package could literally be named `ourlark`, but here are some other ideas:

- `astrolark`
- `stellarjay`
- `greenlark`
- `greylark` (because elephants are grey …)
- `starling`

Reply:
I think starlark is the right name for our stuff on top of starlark.  It is annoying that you often need their package and our package, but this happens a lot in our codebase (src/server/*/server) and it's not particularly problematic.  gopls even suggests what's most frequently used already, so I don't think it's too concerning.  The main idea is for the package to stand on its own, and if there is a little pain when importing, I think it's better than people wondering where the name came from.  ("Did we fork it?")